Modules/mathmodule.c

```diff
@@ -1840,7 +1818,7 @@ math_isqrt(PyObject *module, PyObject *n)
 
 /* factorial_partial_product: Compute product(range(start, stop, 2)) using
  * divide and conquer.  Assumes start and stop are odd and stop > start.
- * max_bits must be >= bit_length(stop - 2). */
+ * max_bits must be >= _Py_bit_length(stop - 2). */
```

Comment:
Could we leave this as `bit_length` in the comments? I think the meaning is clear.

Reply:
Removed the `_Py_` prefix in all comments.